protocol/src/generator.ts

```diff
@@ -247,11 +248,7 @@ function property(name: string, optional: boolean, prop: P.PropertyType): string
 	s += comment(prop);
 	const type = propertyType(prop);
 	const propertyDef = `${name}${optional ? '?' : ''}: ${type}`;
-	if (type[0] === '\'' && type[type.length-1] === '\'' && type.indexOf('|') < 0) {
```

Comment:
Sorry, can you give an example of what changed here?

Reply:
Oh, it's basically all of the type/event/command parameters. It's not clear to me why we are emitting these commented out. And does add a constraint but, for example, if you implemented one of the event types and set `event` different, then that would be wrong and the event wouldn't even be handled correctly. So I feel fine about that change.

Reply:
I think TypeScript will check that the event type (or type, etc.) matches the expected value if you where to try to use the wrong type.